docker/prometheus/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:alpine AS builder
+
+# Install git
+# Git is required for fetching the dependencies.
+RUN apk update && apk add --no-cache git wget
+
+# Install prometheus
+RUN wget https://github.com/prometheus/prometheus/releases/download/v2.22.0/prometheus-2.22.0.linux-amd64.tar.gz
```

Comment:
Not a big deal, but is there a better way to specify the latest release?

Reply:
There's not :(. Normally github has the 'release/latest' download tag but in this case the releases contain multiple architectures *and* the release number is in the filenname.